### ISSUE TITLE
ClassroomDrop-in.zip App Upload

### ISSUE THREAD

**Issue by mrpearce** [OfficeDev/microsoft-teams-apps-classroom-dropin#16]
Hi,

I am trying to upload the zip toe create the app in Power Apps and after connecting it to the correct resources I get the following error?  The following image gives details on the error.  I have the lists inside of the SharePoint site setup.

![Capture](https://user-images.githubusercontent.com/2961612/106327188-970f7380-6275-11eb-8edc-9140d57d01bc.JPG)

**Comment by RobotVoice** [comment]
Yeah i got the same error.  I ending up fixing it my starting over and doing it over. Wish I could say what I did exactly to fix it.

**Comment by SchoolIT886** [comment]
Hi, 

I had the same issue and it was down to Step 4: Import and Set up the Classroom Drop-in
Modifying the PowerApps ZIP Package I wasn't using the right URL or  GUID.

**Comment by mrpearce** [comment]
> Hi,
> 
> I had the same issue and it was down to Step 4: Import and Set up the Classroom Drop-in
> Modifying the PowerApps ZIP Package I wasn't using the right URL or GUID.

I have checked this various times and that still isn't solve the issue.  I cannot find anywhere in the files or flows created a connection that links back to the isOwner as it simply looks like the API they have used is missing the data type that is required for that field.
